### ISSUE TITLE
[BUG]: Database File Not Found

### ISSUE THREAD

**Issue by JeerusBeerus** [Depressurizer/Depressurizer#267]
**Describe the bug**
When attempting to launch Depressurizer to categorize my Steam library, the program fails and throws multiple errors. Below are the error messages encountered:

Error 1: Database File Not Found

Message: "Error loading game database. Local category and DLC data will not be available. Database file does not exist."
Error 2: Invalid VDF Format

Message: "Error encountered when updating database from AppInfo: Invalid VDF format."
Error 3: SSL/TLS Secure Channel Issue

Message: "Error encountered when updating database from HLTB: The request was aborted: Could not create SSL/TLS secure channel."
Error 4: Game Data Not Found

Message: "No game data found. Please make sure the custom URL name is spelled correctly."

**To Reproduce**
1. Start the program
2. See it fail.

**Expected behavior**
The program should load the Steam game database without errors and allow categorization of the library.

**Actual Behavior** 
The program displays multiple errors related to missing files, invalid formats, and connection issues.

**Screenshots**
![image](https://github.com/user-attachments/assets/d306f657-6451-430b-a7ae-6e78b447058a)
![image](https://github.com/user-attachments/assets/b32198ea-b79a-47dd-b648-8b24e18d8b4c)
![image](https://github.com/user-attachments/assets/4c9105da-012a-475d-86c9-d5eb0198e2c2)
![image](https://github.com/user-attachments/assets/9d398e0e-c2e6-4233-a996-74bc76411bda)
![image](https://github.com/user-attachments/assets/8e101f4b-0526-48d2-baf0-8d1b298ce16e)

**Desktop (please complete the following information):**
 - OS: Windows 11 Pro
 - Version 22631.4037
 -  .NET Version 6.0.23

**Additional context**
Installation folder is correct. Steam API key is correct. I somehow can't make it work.

**Comment by LearnTheNext** [comment]
Steam appears to have changed how/where the library is stored. Depressurizer stopped working for me on beta a while ago, but I got a couple extra months of use by turning it off. Looks like this tool is completely dead now until/unless someone does some legwork. Unfortunately, the project appears to have been abandoned by its creator.

**Comment by jandersonthegreat** [comment]
so glad categories for steam suck and this tool is now useless too

**Comment by Tythesly** [comment]
Yeah honestly steam is shit and should incorporate the same sorting system
as depressurizer

**Comment by pjmdesi** [comment]
Found this trying to find a way to filter my library by local co-op, a shame this has been abandoned :\
If I knew anything about C (or any variation thereof) I'd see if I could fix the library issue...
